scripts/version-lookup.sh

```diff
@@ -3,18 +3,18 @@
 # Use gnu-tar and gnu-date regardless of whether the OS is Linux
 # or BSD-based.  The correct command will be assigned to TAR and DATE
 # variables.
+# shellcheck disable=SC1091
 source "$(dirname "$(which "$0")")/use-gnu-tar.sh"
 
 SCRIPT_DIR=$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)
-BASE_DIR=$(dirname $SCRIPT_DIR)
+BASE_DIR=$(dirname "$SCRIPT_DIR")
 
 versions() {
-  cat dist-newstyle/cache/plan.json \
-    | jq -r '
+  jq -r '
       ."install-plan"[]
       | select(."pkg-src".repo.uri == "https://input-output-hk.github.io/cardano-haskell-packages")
       | {"pkg-name": ."pkg-name", "pkg-version": ."pkg-version"}
-      | @base64' \
+      | @base64' < dist-newstyle/cache/plan.json \
```

Comment:
```suggestion
      | @base64' \
      dist-newstyle/cache/plan.json \
```
wouldn't that work just fine? One less redirection to read?

Reply:
No, that will not work. `jq` is weird and by default works by reading input from `stdin`.